_includes/templates/summaries/linkedin.html

```diff
@@ -0,0 +1,16 @@
+<h2 class="h4">
+    <span class="cf-icon cf-icon-linkedin-square"></span>
+    Follow us on LinkedIn
+</h2>
+<p class="short-desc">
+    The CFPB is one of the most searched for agencies
+    in the federal government.
+    Connect with us to stay updated on the work
+    we do and new opportunities to be a part of it.
+</p>
+<p class="short-desc">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/9c8ea4dd37896e21eeef696cc7c492ce98e7921c  and https://github.com/cfpb/cfgov-refresh/commit/fe406bb5534f5de096b88ca71e04f7729d656452

Are there any issues with an inline block (link) next to a block (paragraph)? I always feel like blocks should be sibling to blocks.